crates/editor/src/editor.rs

```diff
@@ -4577,6 +4577,7 @@ impl Editor {
                     .size(ui::ButtonSize::None)
                     .icon_color(Color::Muted)
                     .selected(is_active)
+                    .tooltip(|cx| Tooltip::text("Code Actions", cx))
```

Comment:
<img width="358" alt="image" src="https://github.com/zed-industries/zed/assets/2690773/9b4d6872-3bdb-43e8-acca-b81314d9436e">

This looks more interesting than the `Run` tooltip as the icon is not that obvious, but not sure if it's changing things a lot for me.

Reply:
This was the original issue I actually wanted to fix.
I couldn't figure out what this lightning bolt icon meant and went on to fix it. However, thought later that adding more tooltips might be better as a PR for just one tooltip would be kind of bloat.

Never mind though.